umake/frameworks/baseinstaller.py

```diff
@@ -117,6 +118,9 @@ def remove(self):
         if self.icon_filename:
             with suppress(FileNotFoundError):
                 os.remove(get_icon_path(self.icon_filename))
+        if self.add_link is True:
+            with suppress(FileNotFoundError):
+                os.remove(os.path.join(os.path.expanduser("~"), '.bin', self.desktop_filename.split('.')[0]))
```

Comment:
maybe set the path in settings.py rather than copying it here and when creating it? (we can reuse it as well in the tests that way)